pandas/core/base.py

```diff
@@ -990,11 +990,17 @@ def factorize(self, sort=False, na_sentinel=-1):
         from pandas.core.algorithms import factorize
         return factorize(self, sort=sort, na_sentinel=na_sentinel)
 
-    def searchsorted(self, key, side='left'):
-        """ np.ndarray searchsorted compat """
+    def searchsorted(self, key, side='left', sorter=None):
```

Comment:
looks for the Appender decorator and also Substitute decorator
used to propagate doc strings 
good examples in core/generic then usage in core/Frame

eg reindex